namespace-provisioner/customize-installation.md

```diff
@@ -144,106 +141,109 @@ Options if using Controller
+    parameter_prefixes:
+    - tmc.cloud.vmware.com
+    - tap.tanzu.vmware.com
+  ```
 
    **<a id='con-import-overlay'></a>Import overlay secrets**
 
-  `overlay_secrets` is a list of secrets which contains Carvel ytt overlay definitions that are applied to the resources created by the Namespace Provisioner. The secrets are imported to `namespace-provisioner` namespace if it is in another namespace. The `stringData` key of the secret must have .`yaml` or .`yml` suffix.
+  `overlay_secrets` is a list of secrets which contains [Carvel ytt overlay](https://carvel.dev/ytt/docs/latest/lang-ref-ytt-overlay/) definitions that are applied to the resources created by the Namespace Provisioner. The secrets are imported to `namespace-provisioner` namespace if it is in another namespace. 
+  >**Note** The `stringData` key of the secret must have .`yaml` or .`yml` suffix.
 
    Sample secret with overlay to be used:
 
-   ```console
-   cat << EOF | kubectl apply -f -
-   apiVersion: v1
-   kind: Secret
-   metadata:
+  ```yaml
+  cat << EOF | kubectl apply -f -
+  apiVersion: v1
+  kind: Secret
+  metadata:
     name: grype-package-overlay
     namespace: tap-install
     annotations:
       kapp.k14s.io/change-rule: "delete after deleting tap"
-   stringData:
-     package_overlay.yaml: |
-       #@  load("@ytt:overlay", "overlay")
-       #@
-       #@  def matchGrypeScanners(index, left, right):
-       #@    if left["apiVersion"] != "packaging.carvel.dev/v1alpha1" or left["kind"] != "PackageInstall":
-       #@      return False
-       #@    end
-       #@    return "metadata" in left and "name" in left["metadata"] and left["metadata"]["name"].startswith("grype-scanner")
-       #@  end
-
-       #@overlay/match by=matchGrypeScanners, expects="0+"
-       ---
-       metadata:
-         annotations:
-           #@overlay/match expects="0+"
-           ext.packaging.carvel.dev/ytt-paths-from-secret-name.0: my-grype-overlay-secret
+  stringData:
+    grype-package-overlay.yaml: |
+      #@  load("@ytt:overlay", "overlay")
+      #@
+      #@  def matchGrypeScanners(index, left, right):
+      #@    if left["apiVersion"] != "packaging.carvel.dev/v1alpha1" or left["kind"] != "PackageInstall":
+      #@      return False
+      #@    end
+      #@    return "metadata" in left and "name" in left["metadata"] and left["metadata"]["name"].startswith("grype-scanner")
+      #@  end
+
+      #@overlay/match by=matchGrypeScanners, expects="0+"
+      ---
+      metadata:
+        annotations:
+          #@overlay/match expects="0+"
+          ext.packaging.carvel.dev/ytt-paths-from-secret-name.0: my-grype-overlay-secret
+  EOF
   ```
 
    Sample TAP values configuration:
 
-   ```console
-   namespace_provisioner:
+  ```yaml
+  namespace_provisioner:
     controller: true
     overlay_secrets:
     - name: grype-package-overlay
       namespace: tap-install
-      create_export: true
+       create_export: true
   ```
 
 Options if using GitOps
```

Comment:
should this have a line break? the header `Options if using Controller` has a new line.